recipes/pdfshrink/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pdfshrink" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8378ea2b46e28f352a74bf8a673a5d9287c70bde0c12e66078a93a3f0357849a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/pdfshrink/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - pdfshrink = pdfshrink:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
```

Comment:
Are `setuptools` and `docopt` really needed here? Usually `python >=3` and `pip` are enough for the `host` section.

Reply:
Done! Thanks!